### ISSUE TITLE
✨ Enhancement: Adding hover effect on the buttons. 

### ISSUE THREAD

**Issue by DhairyaMajmudar** [json-schema-org/website#428]
### Is your feature request related to a problem? Please describe

Adding a hover effect on the buttons of the website make it more visually applying and design-friendly 

### Describe the solution you'd like

Adding hover effect on the buttons with the help of `hover:` tailwind utility classes  

### Describe alternatives you've considered

n/a

### Additional context

n/a

### Are you working on this?

Yes

**Comment by benjagm** [comment]
Thanks @DhairyaMajmudar ! Good catch. Issue accepted.

**Comment by Adity20** [comment]
Can I work on this or are you working on this one @DhairyaMajmudar ?

**Comment by DhairyaMajmudar** [comment]
> Can I work on this or are you working on this one @DhairyaMajmudar ?

Yes @Adity20 can you pls. look for an another issue since I am working on this one

**Comment by Adity20** [comment]
Okay no issue @DhairyaMajmudar.

**Comment by benjagm** [comment]
@DhairyaMajmudar I didn't assigned the issue yet. Please, wait next time.

**Comment by VivekJaiswal18** [comment]
It was showing no Assignees and the issue was very small thats why i fixed it.

**Comment by benjagm** [comment]
@DhairyaMajmudar is it ok with you if we use @VivekJaiswal18 solution?

@VivekJaiswal18 please ... next time wait to be assigned before start working on an issue.

**Comment by DhairyaMajmudar** [comment]
> Se we have 2 PRs right? Please check your colleague solution and let me know what PR do you prefer me to review.

Nope as of now I have done changes in my local branch only, so currently there exists 1 pr , but yes I would like to review that pr : )

**Comment by VivekJaiswal18** [comment]
> @VivekJaiswal18 please ... next time wait to be assigned before start working on an issue.
Okay will keep in mind from next time..there was some misunderstanding from my side related to the assignment.
